includes/Core/Authentication/Verification_Tag.php

```diff
@@ -0,0 +1,123 @@
+<?php
+/**
+ * Class Google\Site_Kit\Core\Authentication\Verification_Tag
+ *
+ * @package   Google\Site_Kit
+ * @copyright 2019 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Authentication;
```

Comment:
This file was removed in the meantime, can you re-remove it?

Reply:
Removed. I've added you as a collaborator to my repo @felixarntz . Let me know if you need anything else, please. Thank you!